django_rq/views.py

```diff
@@ -49,6 +58,25 @@ def stats_json(request, token=None):
     )
 
 
+@never_cache
+@staff_member_required
+def prometheus_metrics(request):
```

Comment:
Mind moving this to a different view file? (probably `stats_views.py`). Since `views.py` is becoming large, I want to split the current `views.py` into `workers_view.py`, `queues_views.py` etc.